### PR TITLE
PCM-1667

### PR DESCRIPTION
**Pull request by maxwellmooney13**

### CODE REVIEW

**Review by djhouseknecht — CHANGES_REQUESTED** [review]
Two things: 

1. It may be good to start a `CHANGELOG.md` since this is open source. I know these are "soft" releases, but it will still be good to have one
2. If you are trying to get another npm version published, you may want to do a patch version bump.

**Comment by zservies** [comment]
> 

I second creating a `CHANGELOG.md`

**Review by jensengar — APPROVED** [review]

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+The format is based on [Keepa Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+# [Unreleased](https://github.com/purecloudlabs/softphone-vendor-headsets/compare/master...HEAD)
+
+# [v0.1.1](https://github.com/purecloudlabs/softphone-vendor-headsets/compare/d230063938501788fff660924cb1f530c1685499...c04b380d08681af57f238d1db170ebd465a3e035)
```
Comment (jensengar) [inline changelog.md:8]: I prefer to do tags over commits. See the webrtc-sdk.

Comment (djhouseknecht) [reply changelog.md:8]: 2nd

**Review by zservies — APPROVED** [review]

**Review by djhouseknecht — APPROVED** [review]

**Comment by djhouseknecht** [comment]
Could you squash the commits when you merge?

**Pull request merged**